wpilibj/src/main/java/edu/wpi/first/wpilibj/command/InstantCommand.java

```diff
@@ -42,8 +44,60 @@ public InstantCommand(String name, Subsystem subsystem) {
     super(name, subsystem);
   }
 
+  /**
+   * Creates a new {@link InstantCommand InstantCommand}.
```

Comment:
For doxygen and javadoc comments, there should be an empty comment line between the description and the @ tags.